### ISSUE TITLE
a11y: Shopping cart keyboard trap

### ISSUE THREAD

**Issue by shadeed** [smashingmagazine/redesign#154]
**- Do you want to request a *feature* or report a *bug*?**
Bug.

**- What is the current behavior?**
1. Moving between the cart links/buttons with `tab` key is causing the focus to go outside. Example: I can focus on one of the navigation links, while the cart modal is active.

<img width="1398" alt="screen shot 2017-03-21 at 8 53 51 am" src="https://cloud.githubusercontent.com/assets/8038659/24135595/6882af60-0e13-11e7-9230-fff5cd09b721.png">

2. The "X" button and cart item are not focusable for some reason.

**- What is the expected behavior?**
Focus should only stay inside the cart, and loop through all links/buttons.

**Comment by dstorey** [comment]
The focus style is also broken on this for the book title (and is another place where there are two consecutive links going to the same place).

Along with the keyboard trap described above the keyboard focus should be on the X button when the dialog opens so that it is easy to close again. Next tab should probably cycle around to the clear cart button. Shift tab should go back to checkout button. It may be better if x was in top left to have more logical flow through with keyboard and screen reader, but that would require redesign rather than simple fix.
